cpp/src/io/parquet/reader_impl.cu

```diff
@@ -461,8 +469,10 @@ class aggregate_reader_metadata {
    */
   [[nodiscard]] std::string get_pandas_index() const
   {
-    auto it = agg_keyval_map.find("pandas");
-    if (it != agg_keyval_map.end()) {
+    // Assumes that all input files have the same metadata
+    // TODO: verify this assumption
+    auto it = keyval_maps[0].find("pandas");
+    if (it != keyval_maps[0].end()) {
```

Comment:
Not sure how it works if the first file is written by pandas (thus has `index_columns`) while the second one is written by cudf (has no `index_columns`). To be verified once the python changes have been finished.